src/App.tsx

```diff
@@ -1,7 +1,36 @@
+//  import { prependOnceListener } from 'process';
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
```

Comment:
if you use the same code more than once - it's make sense to create serapate variable for it
move callback in the separate variable